r/tests/testthat/test-dplyr-glimpse.R

```diff
@@ -15,6 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# For some reason, the glimpse output is different on 3.5
```

Comment:
Do we want something explaining a why?
```suggestion
# For some reason, the glimpse output is different on 3.5
# outside the purview of arrow and more for dplyr to worry about
```

Reply:
I really don't understand why. It's possibly troubling, the data for the boolean column is different in the glimpse: https://github.com/ursacomputing/crossbow/runs/7331660996?check_suite_focus=true#step:5:22363

Reply:
Ah I see why now: the example data uses `sample()` for the logical column and the RNG changed a few releases back: https://github.com/apache/arrow/blob/master/r/tests/testthat/helper-data.R#L22

Reply:
```suggestion
# The glimpse output for tests with `example_data` is different on R < 3.5
# because the `lgl` column is generated with `sample()` and the RNG
# algorithm is different in older R versions. 
```